packages/a11y-base/test/field-aria-controller.test.js

```diff
@@ -158,12 +158,12 @@ const runTests = (defineHelper, baseMixin) => {
         controller.setTarget(element);
       });
 
-      it('should add error id to aria-labelledby attribute', () => {
-        expect(element.getAttribute('aria-labelledby')).equal('custom-id error-id');
+      it('should add error id to aria-describedby attribute', () => {
+        expect(element.getAttribute('aria-describedby')).equal('custom-id error-id');
       });
 
       it('should not add error id to aria-describedby attribute', () => {
```

Comment:
`should not add error id to aria-labelledby attribute`?

Reply:
Yeah. This should've been removed, but accidentally I left it in place.